### PR TITLE
Event Logo cut-off with long event names fixed

### LINKED ISSUE #8029: Event logo cut off in events with long title or address
When an event has a long title or address the information can result in an additional line. In this case the logo moves to the top of the information and this results in logos getting cut off  on some public pages, e.g. speaker pages compare https://eventyay.com/e/0db87d3b/speakers .

Expected is that that the logo stays on the left and the text stays visible as well.

Screenshot from 2021-11-04 15-21-42
Screenshot from 2021-11-04 15-20-03
bu

### PR DESCRIPTION
**Pull request by Ruchip16**
Fixes #8029 

#### Short description of what this resolves:

Event logo with long names used to get cut-off while we scroll down the page

#### Changes proposed in this pull request:

Gave some styling so that text would appear like this

image

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by progmatic-99 — APPROVED** [review]

**Comment by maze-runnar** [comment]
@Ruchip16 not working for mobile view - 
Screenshot from 2021-12-05 12-41-15

**Comment by Ruchip16** [comment]
> @Ruchip16 not working for mobile view - Screenshot from 2021-12-05 12-41-15

yaa, coz I have only changed for web app and not for mobile view coz for that particular example it was working fine for mobile view hence did not pay much attention there

**Review by pc-beast — CHANGES_REQUESTED** [review]
Please make necessary changes for the mobile view too. Also please use helper classes for margins from ``_utils.scss``` If defining a class is a necessity then define it there.

**Comment by pc-beast** [comment]
@Ruchip16 updates?

**Comment by Ruchip16** [comment]
> @Ruchip16 updates?

ya, I haven't made the changes yet but will probably make them tonight and would push the commit here

**Comment by Ruchip16** [comment]
> Please make necessary changes for the mobile view too. Also please use helper classes for margins from ``_utils.scss``` If defining a class is a necessity then define it there.

where can I find the media queries for the particular class coz I am not able to find them in the styles folder

**Comment by Ruchip16** [comment]
> > Please make necessary changes for the mobile view too. Also please use helper classes for margins from ``_utils.scss``` If defining a class is a necessity then define it there.
> 
> where can I find the media queries for the particular class coz I am not able to find them in the styles folder @pc-beast



> Please make necessary changes for the mobile view too. Also please use helper classes for margins from ``_utils.scss``` If defining a class is a necessity then define it there.

Also I have added margin for the ```mr-8 class``` which was defined already in ```utils.scss``` but if I have to add margin or padding to the main div class ```ui info d-flex``` or sub headings class like ```event time ``` , ```event name text-white``` , ```event location``` will it be fine if I define it there or like it has to be defined elsewhere/ already defined elsewhere or just inline CSS would work like I have added it already because I went through entire styles folder in ```app/styles``` dir but I couldn't find there

**Comment by pc-beast** [comment]
just define classes if you are not able to find one according to your need. Many of the classes used are from SemanticUI as we are using it. Please see if can be solved with classes from semantic UI. If not define your own classes in related files. Try to complete it asap as it is a small issue and it should not become a blocker.

**Comment by Ruchip16** [comment]
> just define classes if you are not able to find one according to your need. Many of the classes used are from SemanticUI as we are using it. Please see if can be solved with classes from semantic UI. If not define your own classes in related files. Try to complete it asap as it is a small issue and it should not become a blocker.

yaa okay

File: app/styles/partials/utils.scss [hunk 1]
```diff
@@ -236,4 +236,17 @@
 
 .mr-8 {
   margin-right: 2rem !important;
+  margin-bottom: -120px;
```
Comment (pc-beast) [inline app/styles/partials/utils.scss:239]: mr here means margin-right..do not change anything in existing classes.

File: app/templates/public.hbs [hunk 2]
```diff
@@ -7,10 +7,10 @@
         <div class="ui info d-flex">
           {{#if this.model.logoUrl}}
             <LinkTo @route="public">
-              <img src="{{this.model.logoUrl}}" class="logo mr-8" />
+              <img src="{{this.model.logoUrl}}" class="logo mr-8"/>
             </LinkTo>
           {{/if}}
-          <div>
+          <div style="margin-left:220px">
```
Comment (pc-beast) [inline app/templates/public.hbs:13]: why inline?

File: app/styles/partials/utils.scss [hunk 3]
```diff
@@ -236,4 +236,17 @@
...
+.event.time {
+  margin-left: 220px;
+}
+
+.event.name {
+  margin-left: 220px;
+}
+
+.event.location {
+  margin-left: 220px;
```
Comment (pc-beast) [inline app/styles/partials/utils.scss:251]: these are classes from SemanticUI, do not add properties here

File: app/styles/partials/utils.scss (hunk 1 above)
Comment (Ruchip16) [reply app/styles/partials/utils.scss:239]: okay will revert this

File: app/templates/public.hbs (hunk 2 above)
Comment (Ruchip16) [reply app/templates/public.hbs:13]: Because I knew that those classes are of semantic UI and I did not get any file or folder where the styling of these classes was present hence I did not remove the inline CSS from here

File: app/styles/partials/utils.scss (hunk 3 above)
Comment (Ruchip16) [reply app/styles/partials/utils.scss:251]: ya hence i asked u that where would i get this styling

Comment (Ruchip16) [reply app/styles/partials/utils.scss:251]: > ya hence i asked u that where would i get the styling of these classes

**Review by progmatic-99 — COMMENTED** [review]
@Ruchip16 Please upload a screenshot of your changes in both pc and mobile screen size.

**Comment by Ruchip16** [comment]
> @Ruchip16 Please upload a screenshot of your changes in both pc and mobile screen size.

image

**Comment by progmatic-99** [comment]
@mariobehling pls review once

File: app/styles/partials/utils.scss [hunk 4]
```diff
@@ -159,7 +159,7 @@
 }
 
 .mb-8 {
-  margin-bottom: 2rem !important;
+  margin-bottom: -7.5rem !important;
```
Comment (pc-beast) [inline app/styles/partials/utils.scss:162]: Do not change this definition

**Review by pc-beast — COMMENTED** [review]
Why have you added a module in file changes?

**Comment by Ruchip16** [comment]
> Closing due to inactivity.

🥺🥺🥺🥺

**Comment by pc-beast** [comment]
Feel free to reopen and finish it if you are still working on it.

**Comment by Ruchip16** [comment]
> Feel free to reopen and finish it if you are still working on it.

yeah I will finish it soon, i was stunned for a minute though

**Comment by pc-beast** [comment]
Try to finish it if you are still working on it. Updated the latest changes here. Thanks.

**Comment by Ruchip16** [comment]
> Try to finish it if you are still working on it. Updated the latest changes here. Thanks.

yeahh, actually past week I had some interviews hence did not get time to work on this but will definitely do it soon, sorry for the delay

File: app/templates/public.hbs [hunk 5]
```diff
@@ -7,10 +7,10 @@
         <div class="ui info d-flex">
           {{#if this.model.logoUrl}}
             <LinkTo @route="public">
-              <img src="{{this.model.logoUrl}}" class="logo mr-8" />
+              <img src="{{this.model.logoUrl}}" class="logo mr-8" style="margin-bottom: -120px;"/>
```
Comment (Ruchip16) [inline app/templates/public.hbs:10]: I added inline CSS here coz I couldn't create just another class for the logo as there is one already and plus I had to do changes in margin-bottom and not margin-right

**Review by progmatic-99 — CHANGES_REQUESTED** [review]
Please always upload a ss when you are doing changes to the UI.
Thanks

**Comment by Ruchip16** [comment]
> Existing events do not have the same UI as before.
> 
> **Please test the changes you make with creating a dummy event that tests corner cases and also make sure that existing events stay the same.** Thanks Screenshot 2021-12-22 at 18-57-44 OSBA Members Products eventyay

yaa I noticed that like entire UI gets affected due to that changes but then only inline CSS would prolly help coz otherwise if I create a class like this and do the changes like that it would get changed entirely how should I overcome this?

**Comment by pc-beast** [comment]
> yaa I noticed that like entire UI gets affected due to that changes but then only inline CSS would prolly help coz otherwise if I create a class like this and do the changes like that it would get changed entirely how should I overcome this?

You know you can define a class for everything you do in inline styling, right? Please understand that If we guide you with every word in code, it is better that we implement it ourselves.
